examples/dgmulti_3d/elixir_euler_tensorWedge.jl

```diff
@@ -0,0 +1,59 @@
+using OrdinaryDiffEq
+using Trixi
+using LinearAlgebra
+
+###############################################################################
+equations = LinearScalarAdvectionEquation3D(1.0, 1.0, 1.0)
+
+initial_condition = initial_condition_convergence_test
+
+# Define the polynomial degrees for the polynoms of the triangular base and the line
+# of the tensor-prism
+tensor_polydeg = (2, 3)
+
+dg = DGMulti(element_type = Wedge(),
+            approximation_type = Polynomial(),
+            surface_flux = flux_lax_friedrichs,
+            polydeg = tensor_polydeg)
+
+
+coordinates_min = (-1.0, -1.0, -1.0)
+coordinates_max = (1.0, 1.0, 1.0)
+num_cells = (8, 8, 8)
+cells_per_dimension = num_cells
```

Comment:
Do you need `num_cells` somewhere else? If not, why not setting
```suggestion
cells_per_dimension = (8, 8, 8)
```